pkg/app/piped/cloudprovider/kubernetes/helm.go

```diff
@@ -199,3 +212,70 @@ func (c *Helm) TemplateRemoteChart(ctx context.Context, appName, appDir, namespa
 	}
 	return executor()
 }
+
+// verifyHelmValueFilePath verifies if the path of the values file references
+// a remote URL or inside the path where the application configuration file (i.e. *.pipecd.yaml) is located.
+func verifyHelmValueFilePath(appDir, valueFilePath string) error {
+	url, err := url.Parse(valueFilePath)
+	if err == nil && url.Scheme != "" {
+		for _, s := range allowedURLSchemes {
```

Comment:
was it's a miss or did we really think about allowing the remote valueFiles? I feel a bit unsafe to allow remote file value, wdyt?

Reply:
I understand that this is out of context on this PR 👍 My concern is currently, do we allow the remote valueFiles that way? and if it's yes, then the docs we updated may need to be updated as well.

> Only files stored under the application directory are allowed.

Reply:
Considering that some users are already using the remote values file, I feel it is acceptable to allow it (since it was available before this PR).
I would like to make the following additions to the document, what do you think?

```
Only local files stored under the application directory or remote files served at the http(s) endpoint are allowed.
```

Reply:
okay, lets go with that option 👍

Reply:
updated it :+1:

Reply:
@nghialv Yes, please see this pr: https://github.com/helm/helm/pull/2769

Reply:
Thanks for teaching me. Didn't know about that before. 👍